partcad/src/partcad/adhoc/convert.py

```
@@ -0,0 +1,86 @@
+        "step": "render_step",
+        "brep": "render_brep",
+        "stl": "render_stl",
+        "3mf": "render_3mf",
+        "threejs": "render_threejs",
+        "obj": "render_obj",
+        "gltf": "render_gltf",
+    }
+
+    export_method = export_methods.get(output_type)
+    if not export_method:
+        raise ValueError(f"Unsupported export format: {output_type}")
```

Ruff (0.8.2)

Avoid specifying long messages outside the exception class

(TRY003)